vendor.conf

```diff
@@ -125,7 +125,7 @@ github.com/containerd/ttrpc 94dde388801693c54f88a6596f713b51a8b30b2d
 github.com/gogo/googleapis 08a7655d27152912db7aaf4f983275eaf8d128ef
 
 # cluster
-github.com/docker/swarmkit cfa742c8abe6f8e922f6e4e920153c408e7d9c3b
+github.com/docker/swarmkit d7d23d763a2d47ad6e540f81ab3609f6c323e9be
```

Comment:
Full diff: https://github.com/docker/swarmkit/compare/cfa742c8abe6f8e922f6e4e920153c408e7d9c3b...d7d23d763a2d47ad6e540f81ab3609f6c323e9be

relevant changes;

- https://github.com/docker/swarmkit/pull/2729 Add sysctls support (relates to https://github.com/moby/moby/pull/37701)
- https://github.com/docker/swarmkit/pull/2732 Default Address Pool Code Refactor (relates to this PR)